app/services/region_backfill.rb

```diff
@@ -0,0 +1,127 @@
+class RegionBackfill
+  def self.call(*args)
+    new(*args).call
+  end
+
+  attr_reader :logger
+  attr_reader :invalid_counts
+  attr_reader :success_counts
+
+  def initialize(dry_run: true)
+    @dry_run = dry_run
+    @write = !@dry_run
+    @logger = Rails.logger.child(class: self.class.name, dry_run: dry_run)
+    @success_counts = {}
+    @invalid_counts = {}
+  end
+
+  def dry_run?
+    @dry_run
+  end
+
+  def write?
+    @write
+  end
+
+  def call
+    create_region_types
+    create_regions
+    logger.info msg: "complete", success_counts: success_counts, invalid_counts: invalid_counts
+  end
+
+  def create_regions
+    root_type = find_region_type("Root")
+    org_type = find_region_type("Organization")
+    facility_group_type = find_region_type("FacilityGroup")
+    block_type = find_region_type("Block")
+    facility_type = find_region_type("Facility")
+
+    current_country_name = CountryConfig.current[:name]
```

Comment:
We may want to ensure this only can only be run in India (especially if `dry_run = false`), since the Region hierarchy is likely going to be different elsewhere.
Don't want to trigger this accidentally via a migration or manually in other countries.

Reply:
👍 added a fail safe check